api/http/handler/handler.go

```diff
@@ -221,6 +223,8 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.StripPrefix("/api", h.UserHandler).ServeHTTP(w, r)
 	case strings.HasPrefix(r.URL.Path, "/api/ssl"):
 		http.StripPrefix("/api", h.SSLHandler).ServeHTTP(w, r)
+	case strings.HasPrefix(r.URL.Path, "/api/open_amt"):
```

Comment:
I'd like to have a feature flag around this - so there's no chance any amt code is called when the ff is off

if its awkward, we should improve the ff-conditional code (i do think and explicit `if featureEnabled("open-amt") {` is good tho, we want it to be obvious.

Reply:
I agree, but from the lloks from it and like @hookenz mentioned in a different comment, a larger refactor might be needed for that (currently the `Handler` struct has no access to the featureflag config which is stored in the settings.

What do you think about that leaving like it is now (inside the `openamt/handler` and refactor later?